build/deployment.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
```

Comment:
this file doesn't need to be part of kubeedge source code. it is readily available as a kubernetes artifact. See : https://kubernetes.io/docs/tasks/run-application/run-stateless-application-deployment/

Reply:
We will add links instead of unwanted files in following PR's. Thanks